staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -330,6 +330,27 @@ if (Test-Path $nvidiaInstallLogFolder) {
   }
 }
 
+# Collect kubectl information
+Write-Host "Collecting kubectl information"
+function kubectl { c:\k\kubectl.exe --kubeconfig c:\k\config $args }
```

Comment:
We should only collect these info when both `c:\k\kubectl.exe` and `c:\k\config` exist.

Reply:
Added check